app/routes/articles/components/ArticleEditor.js

```diff
@@ -130,10 +137,17 @@ const onSubmit = (
   dispatch,
   { currentUser, isNew, articleId, submitArticle }: Props
 ) => {
+  const params = new URL(data.youtubeUrl).searchParams;
+  const youtubeParams = {};
+  for (const [key, value] of params.entries()) {
+    youtubeParams[key] = value;
+  }
+
   const body = {
     ...(isNew ? {} : { id: articleId }),
     ...(data.cover ? { cover: data.cover } : {}),
     ...normalizeObjectPermissions(data),
+    youtubeParams,
```

Comment:
Isn't it better to save the url, instead of parsing it before sending? You should also remember to make it support initialValues into the form.

Reply:
True, I can fix so that the parsing happens on receiving the url from the API